### PR TITLE
IconLicence: Add component

### PR DESCRIPTION
**Pull request by felixhabib**
Add `IconLicence` to icon suite

**EXAMPLE USAGE:**
```jsx
<IconLicence />
```

### CODE REVIEW

**Comment by michaeltaranto** [comment]
Ah the `licence` vs `license` discussion!

I was expecting `licence`, I think we follow British unless it follows CSS, e.g. backgroundColor.

**Comment by felixhabib** [comment]
@michaeltaranto ahh yep 😄, wasn't even aware of this spelling

File: site/src/App/routes/foundations/iconography/iconsKeywords.ts [hunk 1]
```diff
@@ -51,6 +51,13 @@ export const iconsKeywords: Record<IconName, NonEmptyArray<string>> = {
   IconInvoice: ['bill', 'payment', 'receipt'],
   IconLanguage: ['translate', 'letters'],
   IconLink: ['chain'],
+  IconLicence: [
+    'card',
+    'permit',
+    'identity',
+    'identification',
+    'certification',
```
Comment (michaeltaranto) [inline site/src/App/routes/foundations/iconography/iconsKeywords.ts:59]: 😛 

```suggestion
    'certification',
    'license',
```

Comment (felixhabib) [reply site/src/App/routes/foundations/iconography/iconsKeywords.ts:59]: ahaha

**Review by michaeltaranto — APPROVED** [review]

**Pull request merged**